src/css/style.css

```diff
@@ -97,7 +102,9 @@ body {
   text-decoration: none;
   vertical-align: middle;
   cursor: pointer;
-  padding: 0.375rem 0.75rem;
+  /* padding: 0.8rem 0.75rem; */
```

Comment:
Is not needed anymore?

Reply:
I think no.

Reply:
Sorry for this,,Done.

Reply:
Sorry, seems like you forgot to push your changes on this one?

Reply:
Please erase the comment that says padding
```suggestion
```